.github/workflows/post-pull-request-checks-automation.yml

```diff
@@ -45,9 +48,26 @@ on:
         description: Configuration for the timeout-minutes parameter of the workflow.
 
 jobs:
+  get_latest_workflow:
+    name: Get the latest workflow run number
+    runs-on: ubuntu-latest
+    outputs:
+      latest_run_number: ${{ steps.get_latest_workflow_run_number.outputs.latest_run_number }}
+    steps:
+      - name: Get the latest workflow run number
+        id: get_latest_workflow_run_number
+        run: |
+          LATEST_RUN_NUMBER=$(curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+                             -H "Accept: application/vnd.github.v3+json" \
+                             "https://api.github.com/repos/${{ github.repository }}/actions/runs?event=push" \
+                             | jq '.workflow_runs[0].run_number')
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Prefer%20to%20use%20PowerShell%207%2B%20scripts